.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,52 @@
+name: "Code scanning - action"
+
+# runs scanning on every push, every pull request, and on Saturdays at 10pm.
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 22 * * 6'
+
+jobs:
+  CodeQL-Build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+        # We must fetch at least the immediate parents so that if this is
+        # a pull request then we can checkout the head.
+        fetch-depth: 2
+
+    # If this run was triggered by a pull request event, then checkout
+    # the head of the pull request instead of the merge commit.
+    - run: git checkout HEAD^2
+      if: ${{ github.event_name == 'pull_request' }}
+      
+    # Initializes the CodeQL tools for scanning.
+    - name: Initialize CodeQL
+      uses: github/codeql-action/init@v1
+      # Override language selection by uncommenting this and choosing your languages
+      with:
+        languages: javascript, python
```

Comment:
You'd think that CodeQL could like, ask Github what the languages in the repo are so it wouldn't need to be specified here but OH KAY.